### PR TITLE
(Java) Add 2 Easy array and 1 Medium LinkedList problem solutions

### PR DESCRIPTION
**Pull request by DrewAMitchell**

### CODE REVIEW

File: Spring_2019/Questions/Medium/OddEvenLinkedList.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    headOdd = new ListNode(head.val);
+                    headOCur = headOdd;
+                }
+                else {
+                    headOCur.next = new ListNode(head.val);
+                    headOCur = headOCur.next;
+                }
+            }
+            head = head.next;
+            if(head == null) headOCur.next = headEven;
```
Comment (ynigoreyes) [inline Spring_2019/Questions/Medium/OddEvenLinkedList.java:42]: Maybe I am reading this wrong, but on line 20, you already check for a null head. Is this line really in the right place?

File: Spring_2019/Questions/Easy/RotateArray.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import java.util.Arrays;
+
+/**
+ * @author Drew Mitchell, TTU-ACM (Applied Algorithms)
+ * Problem Source: https://leetcode.com/problems/rotate-array/
+ */
+
+public class RotateArray {
+	public static void main(String[] args) { //Example test case
```
Comment (ynigoreyes) [inline Spring_2019/Questions/Easy/RotateArray.java:9]: Please do test cases for all Algorithms submitted. I would suggest you look into a real testing library for Java like JUnit.

File: Spring_2019/Questions/Easy/PlusOne.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import java.util.Arrays;
```
Comment (ynigoreyes) [inline Spring_2019/Questions/Easy/PlusOne.java:1]: I will submit the Python version of this. I think you may be over complicating this

**Review by ynigoreyes — CHANGES_REQUESTED** [review]

**Comment by ynigoreyes** [comment]
For Plus One....


```python
class Solution:
    def plusOne(self, digits):
        multiplier = 1
        ans = 1 # Plus One

        # 127 == 100 + 20 + 7
        
        for i in reversed(digits): # Grab the least significant bit
            ans += (i * multiplier) # Start multiplying
            multiplier *= 10           # Each time we iterate, we increase our multiplier
        
        return list(map(int, str(ans)))
```

**Comment by ynigoreyes** [comment]
Again, please provide official/proper test cases for all of your algorithms.

File: Spring_2019/Questions/Medium/OddEvenLinkedList.java (hunk 1 above)
Comment (DrewAMitchell) [reply Spring_2019/Questions/Medium/OddEvenLinkedList.java:42]: Fair point, one of those in-the-moment thoughts. The if check can be bypassed altogether outside the for-loop, given that the problem's preconditions are satisfied (at least 1 node given, non null). Just jumped the gun.

**Comment by DrewAMitchell** [comment]
The two ideas on my mind for PlusOne were:
Do without string parsing, and follow through with carrying in the case that the digit is 9 (9+1 = 10, a decimal carry).
With that said, it didn't occur to me that you could serialize it into an integer data type, however doing so would require me to have to break it out by way of strings all over again.

TL;DR, the solution I provided doesn't use Strings as an intermediary, but would be much simpler that way.

**Comment by ynigoreyes** [comment]
@DrewAMitchell What are we going to do about this PR? Do you want to fix it or use mine?

**Review by ynigoreyes — APPROVED** [review]
